fluent-package/Gemfile

```diff
@@ -31,11 +31,16 @@ if ENV["INSTALL_GEM_FROM_LOCAL_REPO"]
   # into managed Gemfile.lock
   source FLUENTD_LOCAL_GEM_REPO do
     gem "fluentd"
+    # Until https://github.com/chef/win32-service/pull/85 is merged,
+    # this fix should be applied to fluent-package not to block starting fluentdwinsvc service on Windows.
+    # See https://github.com/fluent/fluent-package-builder/issues/618
+    gem "win32-service", github: "fluent-plugins-nursery/win32-service", branch: "fluent-package", platforms: [:mingw, :x64_mingw]
   end
 else
   # Lock to specific revision
   git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
   gem "fluentd", github: "fluent/fluentd", ref: FLUENTD_REVISION
+  gem "win32-service", github: "fluent-plugins-nursery/win32-service", branch: "fluent-package", platforms: [:mingw, :x64_mingw]
```

Comment:
Why is this done on both sides of this `if` branch?
Could you please teach me about `ENV["INSTALL_GEM_FROM_LOCAL_REPO"]`?
If this env var is specified, isn't it unnatural to get the gem from GitHub?

Wouldn't it be more natural to do this around L84?